privaterelay/templates/includes/modal.html

```diff
@@ -6,8 +6,8 @@
   <div class="modal modal-content drop-shadow-light">
     <h2 class="modal-headline">{% ftlmsg 'modal-delete-headline' %}</h2>
     <span class="alias-to-delete demi"></span>
-    <p class="modal-message">{% ftlmsg 'modal-delete-warning-recovery-html' attrs='class="alias-to-delete"' email=request.user.email %}</p>
-    <p class="warning-text  delete-tip modal-message">{% ftlmsg 'modal-delete-warning-upgrade' %}</p>
+    <p class="modal-message">{% ftlmsg 'modal-delete-warning-recovery-html' %}</p>
+    <p class="warning-text  delete-tip modal-message">{% ftlmsg 'modal-delete-warning-upgrade' email=current_relay_address %}</p>
```

Comment:
Same here – because we're no longer using this variable, we can remove it from the Fluent string message. 
```suggestion
    <p class="warning-text  delete-tip modal-message">{% ftlmsg 'modal-delete-warning-upgrade' %}</p>
```